src/view/components/Drawer/DrawerContents/index.tsx

```diff
@@ -1,22 +1,31 @@
 import * as React from 'react';
 import { Content, List, Button, Text } from 'native-base';
-import { routes, RouteInfo, Route } from '../../../navigation/navigator';
+
+import { routesInfo, RouteInfo, Route, DrawerRouteInfo } from '../../../navigation/navigator';
 import style from './style';
 import DrawerItem from '../DrawerItem';
 import DrawerHeader from '../DrawerHeader';
 
 /** DrawerContents props */
-export interface OwnProps {
-  onPress(route: Route): void;
+export interface OwnPropsType {
+  name: string;
+  username: string;
+  onPress: OnPress;
   onLogout(): void;
 }
 
-const DrawerContents: React.StatelessComponent<OwnProps> = (props) => {
+type OnPress = (route: Route) => void;
+
+/**
```

Comment:
did this comment need to be here? im still not sure when we're supposed to be giving function documentation

Reply:
Is it just empty? Oops. It doesn't really need to be there I guess but it is being exported